manifests/plugin/host_reports.pp

```diff
@@ -0,0 +1,58 @@
+# = Foreman Proxy Host Reports plugin
+#
+# This class installs the Host Reports plugin
+#
+# === Parameters:
+#
+# $proxy_name::               Proxy hostname to appear in reports JSON
```

Comment:
Shouldn't this be an advanced parameter? I wonder in which case we actually do want this to be different than the registered name.

For context, the "standard" parameters section is shown with `foreman-installer --help` and advanced parameters are only shown with `foreman-installer --full-help`.

Reply:
I don't have strong opinion here, but I think it's safer even for the user to not change that unless they know what they are doing. But in that case it's easy to change in the config itself.